crates/bevy_gltf/src/loader.rs

```diff
@@ -309,6 +304,23 @@ async fn load_gltf<'a, 'b>(
                 }
             }
 
+            if let Some(vertex_attribute) = reader
+                .read_tangents()
+                .map(|v| VertexAttributeValues::Float32x4(v.collect()))
+            {
+                mesh.insert_attribute(Mesh::ATTRIBUTE_TANGENT, vertex_attribute);
+            } else if primitive.material().normal_texture().is_some() {
```

Comment:
Should this test if the actual mesh has a normal attribute, or is it fine to only test if the source gltf has it?

Reply:
Good catch, I think this should check that both the mesh has a normal attribute and the material has a normal map texture, and in that case, if the mesh does not have a tangent attribute, generate the tangents. Done.